test/index.js

```diff
@@ -73,36 +84,9 @@ describe('sequentially promises', () => {
 
     Promise.resolve(
       promiseq(promises)
-      .catch(() => {
-        /*
-          This catch block exists in order to force the '.then' of the wrapper promise
-          (Promise.resolve(...) above ) to execute. If the behavior this test is protecting against
-          occurs, 'promiseq' will continue calling promises in sequence even after this
-          catch block was executed. After 'promiseq' is resolved, the expectations can be tested
-          inside the next '.then' block. Without this catch block, the catch block of
-          Promise.resolve(...) should be called immeditely, but it would not be possible to detect
-          if anything was executed afterwards.
-
-          Underneath the hood, promise-sequential uses a reduce function which originally called
-          every promise in the sequence in the following way:
-
-          [ () => updateResolvedPromises('p1').catch().then()
-            () => Promise.reject().catch(() => reject()).then()   <--- Rejects 'promiseq' but reduce function not interrupted
-            () => updateResolvedPromises('p3').catch().then()     <--- will execute anyway
-          ].reduce(...)
-            .then()
-
-          Instead, as soon as the first promise is rejected, the error thrown should bail out
-          by executing the 'promiseq' catch block outside of the reduce. E.g.
-
-          [ () => updateResolvedPromises('p1')
-            () => Promise.reject())
-            () => .then(updateResolvedPromises('p3'))
-          ].reduce(...)
-              .then()
-              .catch(() => reject())  <-- after first reject, reduce will stop execution
-        */
-      })
+      /* The catch below prevents the '.catch' of the outer promise, thus the next '.then' will
```

Comment:
it actually does not "prevent" the catch of the outer promise.  We are depending on that to run so that the outer promise is rejected.